### ISSUE TITLE
Add lung segmentation mode

### ISSUE THREAD

**Issue by octavifs** [octavifs/lucanode#1]
Currently the preprocessing for the nodule detection assumes that the lungs will have already been segmented. This is the case for LUNA (as the masks are available) but not so for other datasets.

If we want to extend the usefulness of lucanode to other datasets, it should be able to either to automatically segment lungs.

To this effect, I will employ the same UNET and lung masks already available to train another segmentation neural network. The output of this NN will then be used to perform the nodule segmentation.

**Comment by octavifs** [comment]
Implemented as of 3ed560a. Now training the network for 10 epochs.
